client/rest/test/db/utils/dbTestUtils.js

```diff
@@ -162,7 +162,7 @@ const createDbTransaction = (id, signerPublicKey, recipientAddress, options) =>
 		message: { size: 12, payload: new Binary(test.random.bytes(12)) },
 		mosaics: []
 	};
-	for (let j = 0; 3 < j; ++j)
+	for (let j = 0; 3 > j; ++j)
```

Comment:
what is purpose of this change? 🤔

Reply:
Sorry, this is a mistake. It has been restored to its original state.